site/archive/ArchiveNavigation.tsx

```diff
@@ -0,0 +1,269 @@
+import {
+    ArchivalDateString,
+    type ArchiveSiteNavigationInfo,
+    ArchiveVersions,
+} from "@ourworldindata/types"
+import { PROD_URL } from "../SiteConstants.js"
+import { useMemo, useState } from "react"
+import * as R from "remeda"
+import { useWindowQueryParams } from "../hooks.js"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { useVersionsQuery } from "./queries.js"
+import { VersionsDrawer } from "./VersionsDrawer.js"
+import { useArchiveVersions } from "./versions.js"
+
+export const DATE_FORMAT = "MMMM D, YYYY"
+export const TIME_FORMAT = "HH:mm"
+
+export interface ArchiveNavigationProps
+    extends Omit<ArchiveSiteNavigationInfo, "contentType"> {
+    archivalDate: ArchivalDateString
+}
+
+type ArchiveArticleNavigationProps = Omit<
+    ArchiveNavigationProps,
+    "previousVersion" | "nextVersion"
+>
+
+function ArchiveText({ tooltipContent }: { tooltipContent: string }) {
+    return (
+        <div className="archive-navigation-bar__archive_text">
+            Archive
+            <Tippy
+                content={tooltipContent}
+                theme="owid-footnote"
+                placement="right"
+                appendTo={() => document.body}
+                trigger="mouseenter focus click"
+                interactive
+                interactiveDebounce={50}
+            >
+                <span className="archive-navigation-bar__archive_text--tooltip">
+                    <FontAwesomeIcon icon={faInfoCircle} />
+                </span>
+            </Tippy>
+        </div>
+    )
+}
+
+function ArchiveDate({ dayjsDate }: { dayjsDate: dayjs.Dayjs }) {
+    return (
+        <div className="archive-navigation-bar__date">
+            <span className="archive-navigation-bar__date-value">
+                {dayjsDate.format(DATE_FORMAT)}
+            </span>
+            &nbsp;
+            <span className="archive-navigation-bar__time-value">
+                at {dayjsDate.format(TIME_FORMAT)} UTC
+            </span>
+        </div>
+    )
+}
+
+export function ArchiveChartNavigation(props: ArchiveNavigationProps) {
+    return (
+        <div className="archive-site-navigation">
+            <ArchiveHeaderLogoBar />
+            <ArchiveChartNavigationBar {...props} />
+        </div>
+    )
+}
+
+function ArchiveHeaderLogoBar() {
+    return (
+        <div className="archive-header-logo-bar">
+            <div className="wrapper">
+                <SiteLogos homeUrl={PROD_URL} />
+            </div>
+        </div>
+    )
+}
+
+function ArchiveChartNavigationBar({
+    liveUrl,
+    previousVersion: propsPreviousVersion,
+    nextVersion: propsNextVersion,
+    archivalDate,
+    versionsFileUrl,
+}: ArchiveNavigationProps) {
+    const hasButtons = !!(liveUrl || propsPreviousVersion || propsNextVersion)
+    const dayjsDate = parseArchivalDate(archivalDate)
+    const pageArchivalDate = formatAsArchivalDate(dayjsDate)
+    const { data: versions } = useVersionsQuery(versionsFileUrl)
```

Comment:
If we don't do the sorting in `useArchiveVersions`, we can use it here directly.

Reply:
Makes sense, done.